src/tokenizer/mod.rs

```diff
@@ -290,6 +290,46 @@ pub mod test {
         assert_token(&tokens[3], 0, "frank", 0, 5);
     }
 
+    #[test]
+    fn test_ngram_non_ascii_tokenizer() {
```

Comment:
please add a test where codepoint != grapheme.

You can use this string for instance.
https://play.rust-lang.org/?gist=4ca51df298f3e2d1008872bb0b2b503d&version=stable&mode=debug&edition=2015

Reply:
I think it is ok to not handle grapheme for this functionality as long as we document it and we have a unit test for it.